frontend/packages/dev-console/console-extensions.json

```
@@ -186,6 +186,30 @@
       ]
     }
   },
+  {
+    "type": "dev-console.add/action-group",
+    "properties": {
+      "id": "sharing",
+      "name": "%devconsole~Sharing%",
+      "insertAfter": "helm-chart-repositories",
+      "insertBefore": "local-machine"
+    }
+  },
+  {
+    "type": "dev-console.add/action",
+    "properties": {
+      "id": "project-access",
+      "groupId": "sharing",
+      "href": "/project-access/ns/:namespace",
+      "label": "%devconsole~Sharing%",
+      "description": "%devconsole~Project access allows you to add or remove a user's access to the project.%",
```

nit: for the consistency can we remove the full stop (.) As we do not have it for any descriptions of the add action on add page.
```suggestion
      "description": "%devconsole~Project access allows you to add or remove a user's access to the project%",
```